fourth_task/src/Letter.ts

```diff
@@ -1,7 +1,23 @@
-import { Shipment } from './Shipment';
+import { ShipmentInterface } from './Shipment';
+
+export class Letter implements ShipmentInterface {
+  shipmentId: number;
```

Comment:
@dianalemen, what purpose of deploying all these fields?

Reply:
@AlexHladin what do you mean by deploying?

Reply:
@AlexHladin, sorry, I meant "adding"

Reply:
@AlexHladin no worries
es lint requires to add these fields if we want to add them to `this` in the constructor